plugin/dashboard.lua

```diff
@@ -1,14 +1,22 @@
 -- version 0.2.3
-if vim.g.loaded_dashboard then
-  return
-end
 
-vim.g.loaded_dashboard = 1
+vim.api.nvim_create_augroup('dashboard', { clear = true })
+
+vim.api.nvim_create_autocmd('StdinReadPre', {
```

Comment:
use StdinReadPre how the startup time take ?

Reply:
![wezterm-gui_NhQ5wDDQkC](https://github.com/nvimdev/dashboard-nvim/assets/57625126/cf384dd5-55b9-4e6a-a81d-4cf0cbadb9f8)

master
- 14.019
- 13.470
- 14.572
- 13.177
- 14.534

![wezterm-gui_IpOp9sPI73](https://github.com/nvimdev/dashboard-nvim/assets/57625126/4983461d-a9d0-4814-9c1a-def0db73985a)

pr
- 13.668
- 14.700
- 12.355
- 18.011
- 12.677

Reply:
I think you can just use --startuptime. and i am plan to add new api in core to get startuptime